tests/e2e/install_test.go

```diff
@@ -128,7 +128,11 @@ var _ = Describe("E2E - Install Rancher Manager", Label("install"), func() {
 
 			// Set specified version if needed
 			if rancherVersion != "" && rancherVersion != "latest" {
-				flags = append(flags, "--version", rancherVersion)
+				if rancherVersion == "devel" {
+					flags = append(flags, "--devel")
```

Comment:
I dont think this are exclusive no? You can set --devel AND a version, only --devel will inlcude in the list all the devel versions.

Maybe it will be useful if the latest devel is broken but we need to test with the previous devel version?

Reply:
`--devel` isn't needed if you specified a version. We tried this morning and you can set `--version 2.7.0.-rc7` and it will install this development/RC version without the need of `--devel`. `--devel` is useful here to force the installation of the latest development version.

So with the variables in the workflow files you can set this if you always want the latest version (which is needed for UI):
```
with:
  rancher_channel: latest
  rancher_version: devel
```

And this if you want a specific development version:
```
with:
  rancher_channel: latest
  rancher_version: 2.7.0-rc4
```